### DISCUSSION TITLE
(how) is this different from CERN Root?

### DISCUSSION THREAD

**Discussion by aaarbk** [vpand/icpp#7] [General]
This project looks really interesting - planning to check it out soon!

Was just wondering what the differences are between this and the C++ interpreter that comes with [CERN Root](https://root.cern/), in case anyone is aware. 

I realise they (i.e. CERN) position their software as a "data analysis" framework, so that's one difference already, but the project includes a [C++ interpreter](https://root.cern.ch/root/htmldoc/guides/primer/ROOTPrimer.html#learn-c-at-the-root-prompt) which works reasonably well.

**Comment by GeekNeo** [comment]
The icpp [wiki](https://github.com/vpand/icpp/wiki/) could answer your question.
